lib/esaebsad/esaebsadcommand.rb

```diff
@@ -1,8 +1,11 @@
-class ESAEBSADCommand 
-  
-  $sub_classes = []
-  
+class ESAEBSADCommand
+  @sub_classes = []
+
   def self.inherited(sub_class)
-    $sub_classes.push(sub_class)
+    @sub_classes.push(sub_class)
```

Comment:
That should be changed to << btw (not really anything related to you, but whatever)